example_experiments/simple_experiment.json

```diff
@@ -6,6 +6,10 @@
             "type": "bow"
         }
     },
+    "pretrained_embeddings_file": "filename to gzipped glove vectors get from efs/data/dlfa/glove.840B.300d.txt.gz",
```

Comment:
I wouldn't check this in with a fake path, or with the comment below.  If you want the simple experiment to have these params as an example (which I think is fine), it should be with params that actually work if you run this from a machine with the EFS drive.  So just change this to the correct path.